packages/node/src/types.ts

```diff
@@ -85,14 +85,9 @@ export interface ApiCallTemplate {
   readonly id: string;
 }
 
-export interface Withdrawal {
-  readonly airnodeAddress: string;
-  readonly sponsorAddress: string;
-}
-
 export interface GroupedRequests {
   readonly apiCalls: Request<ApiCall>[];
-  readonly withdrawals: Request<Withdrawal>[];
+  readonly withdrawals: Request<{}>[];
```

Comment:
I'm not quite sure I understand this. Why not just make separate `Withdrawal` interface instead? Is this in anticipation of refactoring later?

Reply:
The only reason that I found for not doing that is because I got a linting error complaining about an empty interface.

Reply:
Even though I like [this](https://github.com/api3dao/airnode/pull/559#issuecomment-921163176) idea the changes I had to make were way too many in comparison with the benefits of that approach so I ended up adding an empty `Withdrawal` interface with a `es-lint-disable-next-line`

Reply:
:/ Mhh, why did it cause so many changes? 

I think is simpler if it is done like in https://github.com/api3dao/airnode/pull/559#issuecomment-921163176

Reply:
Maybe I did something wrong but these types are used all over in the node package. When I finished I had over 25 files modified and there were still some compilation errors so at this point I realized that the change was not worth the trouble.